modules/katamari/src/test/ts/atomic/api/arr/ArrGetTest.ts

```diff
@@ -1,40 +1,42 @@
-import { Assert, UnitTest } from '@ephox/bedrock-client';
-import { Testable } from '@ephox/dispute';
+import { context, describe } from '@ephox/bedrock-client';
 import fc from 'fast-check';
 import * as Arr from 'ephox/katamari/api/Arr';
-import { Optional } from 'ephox/katamari/api/Optional';
-import { tOptional } from 'ephox/katamari/api/OptionalInstances';
+import { assertNone, assertSome } from 'ephox/katamari/test/AssertOptional';
 
-const { tNumber } = Testable;
+describe('atomic.katamari.api.arr.ArrGetTest', () => {
+  context('Arr.get', () => {
 
-UnitTest.test('Arr.get: empty', () => {
-  Assert.eq('none', Optional.none<number>(), Arr.get<number>([ ], 0), tOptional(tNumber));
-});
+    describe('Arr.get: empty', () => {
```

Comment:
```suggestion
    it('Arr.get: empty', () => {
```
And the rest of the file by the looks

Reply:
fixed